<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="grid_placeholder">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>1631</width>
      <height>861</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/home button.png</normaloff>../Images/home button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_button">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>30</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Images/reload button.png</normaloff>../Images/reload button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>50</y>
      <width>102</width>
      <height>901</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="ClickableLabel" name="white_ant_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_ant_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_ant_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_grasshopper_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_grasshopper_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_grasshopper_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_beetle_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Beetle.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_beetle_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Beetle.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_spider_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Spider.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_spider_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Spider.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="white_bee">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/White Bee.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>102</width>
      <height>901</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="ClickableLabel" name="black_ant_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_ant_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_ant_3">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Ant.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_grasshopper_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_grasshopper_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_grasshopper_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Grasshopper.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_beetle_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Beetle.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_beetle_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Beetle.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_spider_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Spider.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_spider_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Black Spider.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ClickableLabel" name="black_bee">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Images/Blace Bee.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>GUI.Src.ClickableLabel</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
